specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/examples/stats/SuccessfulDeleteAssignedAzureAccountRequest.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "Endpoint": "{Endpoint}",
+    "Ocp-Apim-Subscription-Key": "00000000000000000000000000000000",
+    "Authorization": "Bearer {ARM Token}",
+    "appId": "363187f1-c573-46b3-bc4c-ae01d686e68e",
+    "azureAccountInfoObject": {
+      "azureSubscriptionId": "{azure_subscription_id}",
```

Comment:
Did you run OAV on these examples? If not, please run it to validate the swagger file.